daemon/info.go

```diff
@@ -75,7 +75,7 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 		DriverStatus:       daemon.layerStore.DriverStatus(),
 		Plugins:            daemon.showPluginsInfo(),
 		IPv4Forwarding:     !sysInfo.IPv4ForwardingDisabled,
-		BridgeNfIptables:   !sysInfo.BridgeNfCallIptablesDisabled,
+		BridgeNfIptables:   !sysInfo.BridgeNfCallIPtablesDisabled,
```

Comment:
`BridgeNfIptables` above in the `types.Info` struct is still inconsistent with `BridgeNfIP6tables` in that same struct. Should that be fixed too?

I'm actually not sure whether `IP6tables` or `Ip6tables` is more correct here. I could see a case for either (`ip6tables` is a command, so it arguably makes sense to treat it as a word).

Also, being even more pedantic (but hey, this is a PR about character casing): Should the "f" in "BridgeNf" be made capital? I don't know offhand what this means, but it looks like an acronym (for netfilter?).